### PR TITLE
shell questions and answers

### PR DESCRIPTION
**Pull request by vickiemai**
please review and append

### CODE REVIEW

**Review by mikolodian — APPROVED** [review]
Thank you Emmanuel

File: shellqa.txt
Comment (emakono) [inline shellqa.txt:1]: Good job engineer emakono. You shall be billionaire in Jesus name.

**Review by emakono — APPROVED** [review]

**Review by lunaafrique — COMMENTED** [review]
sweet

**Comment by Cheyenne077** [comment]
Approved!

File: shellqa.txt
Comment (Cheyenne077) [reply shellqa.txt:1]: Awesome, great job excellent engineers

**Review by Cheyenne077 — APPROVED** [review]

**Review by Pride22 — COMMENTED** [review]
God blees group4

File: shellqa.txt
Comment (Airmekar89) [reply shellqa.txt:1]: Good Job, Thank you Engineer

**Review by Airmekar89 — APPROVED** [review]

**Review by Pride22 — APPROVED** [review]
God blees group4

**Review by lunaafrique — APPROVED** [review]
Good job guys.

**Comment by vickiemai** [comment]
thank you engineers

**Review by Pride22 — APPROVED** [review]
God bless group4

**Pull request merged**